Pod/UIKit/Classes/UILabel+BonMotUtilities.m

```diff
@@ -7,33 +7,35 @@
 //
 
 #import "BonMot.h"
+#import "BONChain_Private.h"
+
 @import ObjectiveC.runtime;
 
 @implementation UILabel (BonMotUtilities)
 
-- (void)setBonTextable:(id<BONTextable>)textable
+- (void)setBonChain:(BONChain *)chain
 {
-    objc_setAssociatedObject(self, @selector(bonTextable), textable, OBJC_ASSOCIATION_COPY_NONATOMIC);
+    objc_setAssociatedObject(self, @selector(bonChain), chain, OBJC_ASSOCIATION_COPY_NONATOMIC);
 
-    // If the textable is empty, use the label’s existing text
-    if (textable.text.generatesEmptyString) {
+    // If the chain is empty, use the label’s existing text
+    if (chain.generatesEmptyString) {
```

Comment:
It sounds a little too much like magic. I'm going to go with the most basic behavior with the fewest surprises: If you assign a chain, it always clobbers the text. I think this comports with how it would typically be used: assign the chain at setup, and then assign the string multiple times. If you're swapping out the chain, you can just mix in the string while you go. Otherwise, it's too complex to get what I see as the commonly desired behavior. This is going to be a breaking change compared to what we had originally discussed, @imperiopolis, but since this stuff hasn't shipped yet, I'm OK with it. I think @KingOfBrian makes a good point that you can't clear the label with an empty chain.